docs/source/commandline/commandline-all.rst

```diff
@@ -1189,13 +1292,33 @@ Debugging Options
 
 **-[no]update_in_pause**
 
-	Enables updating of the main screen bitmap while the game is paused. This means that the VIDEO_UPDATE callback will be called repeatedly during pause, which can be useful for debugging. The default is OFF (*-noupdate_in_pause*).
+	Enables updating of the main screen bitmap while the system is paused. This means that the VIDEO_UPDATE callback will be called repeatedly during pause, which can be useful for debugging. The default is OFF (*-noupdate_in_pause*).
 
 .. _mame-commandline-watchdog:
 
 **-watchdog** *<duration>* / **-wdog** *<duration>*
 
-	Enables an internal watchdog timer that will automatically kill the MAME process if more than *<duration>* seconds passes without a frame update. Keep in mind that some games sit for a while during load time without updating the screen, so *<duration>* should be long enough to cover that. 10-30 seconds on a modern system should be plenty in general. By default there is no watchdog.
+	Enables an internal watchdog timer that will automatically kill the MAME process if more than *<duration>* seconds passes without a frame update. Keep in mind that some systems sit for a while during load time without updating the screen, so *<duration>* should be long enough to cover that. 10-30 seconds on a modern system should be plenty in general. By default there is no watchdog.
+
+.. _mame-commandline-debuggerfont:
+
+**-debugger_font** *<fontname>* / **-dfont** *<fontname>*
+
+	Specifies the name of the font to use for debugger windows.
+
+	The Windows default font is *Lucida Console*.
+	The Mac (Cocoa) default font is system default.
```

Comment:
System fixed pitch font (typically Monaco).  This is not the same as the system font (typically Lucida Grande).

Reply:
Fixed on my side.